ada_2_fhir-r4/zibs2024/MAPPING_GUIDELINES.md

```diff
@@ -13,66 +13,74 @@ Files and folders are organized in the following way:
 
 Most of the ADA input content can be mapped one-on-one to the FHIR output. For just about every data type, a template is available to transform the ADA element to a FHIR element. See the files `../fhir/2_fhir_fhir_include.xsl` and `../../ada_2_fhir/fhir/2_fhir_fhir_include.xsl`.
 
 ### Specific guidelines
+#### Profile name variable
+For each FHIR profile, add a variable of the form `profileName[English zib name]` to the respective zib stylesheet (in particular, not _in_ the profile template itself). This variable should be used whenever this profile needs to be referenced, instead of stating the profile name directly. This ensures a smaller risk on typos in the profile name and makes it easier to change a profile name in all relevant templates.
+
+Example:
+
+    <xsl:variable name="profileNamePatient">nl-core-Patient</xsl:variable>
+
 #### Parameters
 Each profile template SHOULD have an optional parameter called `$in` that contains the ADA object being transformed, defaulting to the current context.
 
 Each profile template SHOULD have an optional parameter called `$subject` containing the ADA object of the patient to which the zib instance applies. Of course, this doesn't apply when the instance is not about a patient.
 
 Additional parameters may be used as desired, as long as they are documented properly.
 
-#### Profile name variable
-For each FHIR profile add a variable of the form `profileName[English zib name]` to the respective template. This variable should be used whenever this profile needs to be referenced, instead of stating the profile name directly. This ensures a smaller risk on typos in the profile name and makes it easier to change a profile name in all relevant templates.
-
-Example:
-
-    <xsl:variable name="profileNamePatient">nl-core-Patient</xsl:variable>
-
 #### Logical id
-In each template that is used to map an ADA instance to a FHIR resource instance, a logical id should be added to the resulting FHIR instance. This is done by invoking the template `insertLogicalId`. Note that the parameter `profile` may be used whenever there is no one-on-one relation between the ADA element and FHIR resource.
+In each template that is used to map an ADA instance to a FHIR resource instance, a logical id should be added to the resulting FHIR instance. This is done by invoking the template `insertLogicalId`, in which the `profile` parameter SHOULD be used, even though it is only strictly necessary whenever there is no one-on-one relation between the ADA element and FHIR resource.
```

Comment:
Why? This seems like the thing you don't want to do

Reply:
Discussed via Teams: handy to always add the `profile` parameter, to ensure that the logical id is always assigned correctly, even if new relations between the ADA element and FHIR resource would be added in the future.